hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/ReconConstants.java

```diff
@@ -49,6 +49,8 @@ private ReconConstants() {
   public static final String RECON_QUERY_BATCH_PARAM = "batchNum";
   public static final String RECON_QUERY_PREVKEY = "prevKey";
   public static final String PREV_CONTAINER_ID_DEFAULT_VALUE = "0";
+  public static final String PREV_BLOCK_ID_PENDING_FOR_DELETION_DEFAULT_VALUE =
```

Comment:
Suggest rename PREV_BLOCK_ID_PENDING_FOR_DELETION_DEFAULT_VALUE  to 

PREV_DELETED_BLOCKS_TRANSACTION_ID_DEFAULT_VALUE